app/upgrades/v1/mainnet/btcstaking_params.go

```diff
@@ -1,28 +1,114 @@
 package mainnet
 
-// TODO Some default parameters
+// TBD for last parameter version
+// - btc_activation_height, set as 893434 btc block as somewhere in mid april
 const BtcStakingParamsStr = `[
   {
     "covenant_pks": [
-      "43311589af63c2adda04fcd7792c038a05c12a4fe40351b3eb1612ff6b2e5a0e",
-      "d415b187c6e7ce9da46ac888d20df20737d6f16a41639e68ea055311e1535dd9",
-      "d27cd27dbff481bc6fc4aa39dd19405eb6010237784ecba13bab130a4a62df5d",
-      "a3e107fee8879f5cf901161dbf4ff61c252ba5fec6f6407fe81b9453d244c02c",
-      "c45753e856ad0abb06f68947604f11476c157d13b7efd54499eaa0f6918cf716"
+      "d45c70d28f169e1f0c7f4a78e2bc73497afe585b70aa897955989068f3350aaa",
+      "4b15848e495a3a62283daaadb3f458a00859fe48e321f0121ebabbdd6698f9fa",
+      "23b29f89b45f4af41588dcaf0ca572ada32872a88224f311373917f1b37d08d1",
+      "d3c79b99ac4d265c2f97ac11e3232c07a598b020cf56c6f055472c893c0967ae",
+      "8242640732773249312c47ca7bdb50ca79f15f2ecc32b9c83ceebba44fb74df7",
+      "e36200aaa8dce9453567bba108bdc51f7f1174b97a65e4dc4402fc5de779d41c",
+      "cbdd028cfe32c1c1f2d84bfec71e19f92df509bba7b8ad31ca6c1a134fe09204",
+      "f178fcce82f95c524b53b077e6180bd2d779a9057fdff4255a0af95af918cee0",
+      "de13fc96ea6899acbdc5db3afaa683f62fe35b60ff6eb723dad28a11d2b12f8c"
     ],
-    "covenant_quorum": 3,
-    "min_staking_value_sat": 10000,
-    "max_staking_value_sat": 10000000000,
-    "min_staking_time_blocks": 10,
-    "max_staking_time_blocks": 65535,
-    "slashing_pk_script": "dqkUAQEBAQEBAQEBAQEBAQEBAQEBAQGIrA==",
-    "min_slashing_tx_fee_sat": 1000,
-    "slashing_rate": "0.100000000000000000",
-    "unbonding_time_blocks": 101,
-    "unbonding_fee_sat": 1000,
+    "covenant_quorum": 6,
+    "min_staking_value_sat": 500000,
+    "max_staking_value_sat": 5000000,
+    "min_staking_time_blocks": 64000,
+    "max_staking_time_blocks": 64000,
+    "slashing_pk_script": "agdiYWJ5bG9u",
+    "min_slashing_tx_fee_sat": 100000,
+    "slashing_rate": "0.001",
+    "unbonding_time_blocks": 1008,
+    "unbonding_fee_sat": 64000,
```

Comment:
In [the doc](https://babylonlabs.atlassian.net/wiki/spaces/BABYLON/pages/127074712/Phase-2+Mainnet+Parameters) I see this value is `32000`

Reply:
The first btc staking parameters should correspond to the global network parameters https://github.com/babylonlabs-io/networks/blob/main/bbn-1/parameters/global-params.json